editors/vim/Portfile

```diff
@@ -138,51 +138,81 @@ foreach s ${pythons_suffixes} {
     "
 }
 
-variant ruby requires ruby18 description {Compatibility variant, requires +ruby18} {}
-variant ruby18 conflicts ruby19 ruby20 ruby21 ruby22 ruby23 ruby24 ruby25 ruby30 description {Enable Ruby scripting} {
+variant ruby requires ruby33 description {Compatibility variant, requires +ruby33} {}
```

Comment:
I would suggest dropping this +ruby variant. We also used to have a +python, but it only made it more complicated for users to switch to a newer variant on upgrade.

Reply:
I agree this about not presetting a version. I'll be validating `MacVim` port and deleting the variant.